osf/management/commands/import_EGAP.py

```diff
@@ -197,33 +217,40 @@ def main(guid, creator_username):
         project = draft_registration.branched_from
```

Comment:
I'm still a little concerned about working from a query of draft egap registrations. If the EGAP schema is added but not active, does all this still work? If so, I think that might be a good way to ensure that nobody tries to draft register something with the egap schema and suddenly finds they have a real registration.

Reply:
I'll test if it works with an inactive schema

Reply:
Modified the script for everything to happen in a single larger loop so it isn't reliant on retrieving DraftRegistrations with the EGAP_Registration schema